prow/cmd/deck/static/plugin-help.html

```diff
@@ -1,75 +1,12 @@
-                <div class="mdl-layout__header-row">
-                    <a href="https://github.com/kubernetes/test-infra/tree/master/prow#prow" class="logo"><img id="img" src="/logo.svg" alt="kubernetes logo" class="logo"/></a>
-                    <span class="mdl-layout-title header-title">Plugin Help</span>
-                </div>
-            </header>
-            <div class="mdl-layout__drawer">
-                <span class="mdl-layout-title">Prow Dashboard</span>
-                <nav class="mdl-navigation">
-                    <a class="mdl-navigation__link" href="/">Prow Status</a>
-                    <a class="mdl-navigation__link mdl-navigation__link--current" href="/plugin-help.html">Plugin Help</a>
-                    <a class="mdl-navigation__link" href="/tide.html">Tide Status</a>
-                </nav>
-            </div>
-            <main class="mdl-layout__content">
-                <aside>
-                    <div class="card-box">
-                        <ul class="noBullets">
-                            <li>Plugin help for</li>
-                            <li><select id="repo" onchange="redraw();"><option>all plugins</option></select></li>
-                        </ul>
-                    </div>
-                </aside>
-                <table id="command-table" class="mdl-data-table mdl-js-data-table mdl-shadow--2dp plugin-table">
-                    <thead>
-                    <tr>
-                        <th></th>
-                        <th class="mdl-data-table__cell--non-numeric">Command</th>
-                        <th class="mdl-data-table__cell--non-numeric">Example</th>
-                        <th class="mdl-data-table__cell--non-numeric">Description</th>
-                        <th class="mdl-data-table__cell--non-numeric">Who Can Use</th>
-                        <th class="mdl-data-table__cell--non-numeric">Plugin</th>
-                        <th></th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    </tbody>
-                </table>
-            </main>
-        </div>
-        <dialog id="dialog" class="mdl-dialog">
-            <h4 class="mdl-dialog__title"></h4>
-            <div class="mdl-dialog__content">
-            </div>
-            <div class="mdl-dialog__actions">
-                <button type="button" class="mdl-button close">Close</button>
-            </div>
-        </dialog>
-        <div id="toast" class="mdl-js-snackbar mdl-snackbar">
-            <div class="mdl-snackbar__text"></div>
-            <button class="mdl-snackbar__action" type="button"></button>
-        </div>
-    </body>
-</html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="refresh" content="5; url=./command-help.html">
+  <title>Plugin Help</title>
+</head>
+<body>
+  <!-- TODO(@qhuynh96): This is temporary. Should be removed as soon as people adopting to new Command page -->
```

Comment:
realistically we'll never get rid of all links to plugin-help.html, hopefully nobody decides to remove this anytime soon